src/telemetry/telemetry.controller.ts

```diff
@@ -37,11 +69,11 @@ const TELEMETRY_WHITELIST = new Map<string, true | Array<string>>([
       'heap_used',
       'peers_count',
       'session_id',
-      'inbound_traffic',
-      'outbound_traffic',
```

Comment:
Can we bring these back? They're very useful for monitoring network conditions.

Reply:
They should still be present by way of `...traffic_types` below. I can remove that line if it's less clear.